types/console-log-tree/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for console-log-tree 1.2
+// Project: https://github.com/QiShaoXuan/log-tree#readme
+// Definitions by: Rajas Paranjpe <https://github.com/ChocolateLoverRaj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace library;
+
+export interface Tree {
+  name: string;
+  children?: readonly Tree[];
```

Comment:
those can be changed via settings, could bear names:
https://github.com/QiShaoXuan/log-tree/blob/f4e39c4222a4e6caf60ab7f9493eaa1258249ac7/src/index.js#L35-L36
it'd be probably beneficial to define `Settings` interface and `setSettings` method

Reply:
@peterblazejewicz Should I change the library structure to be `class LogTree` and export an instance of `LogTree`? That way people can do `logTree.settings = obj`.

Reply:
thats ok to stay with exported symbols that reassemble a class export.
I don't think 'settings' is a public api, but it's your call here.
You can export it as property:
```ts
export let settings: Settings;
```
or allow to change settings with method:
```ts
export function setSettings(settings?: Settings): void;
```

Reply:
I think it'll be more accurate to just make a class in the types. Then it will be clear that settings is a property and the functions are the prototype.

Reply:
But actually if `settings.keyword.name` and `settings.keyword.children` are changed, the types for `Tree` will be different from the custom keywords. Should I still include `settings`?

Reply:
> I don't think 'settings' is a public api, but it's your call here.

@peterblazejewicz @amcasey I've decided to not include `settings`, because it will mess up the the `Tree` type.